api/rbdigital.py

```diff
@@ -704,20 +704,29 @@ def patron_remote_identifier(self, patron):
         # Set up a refresher method that takes no arguments except the
         # credential -- this is what lookup() expects.
         def refresh_credential(credential):
-            remote_identifier = self._find_or_create_remote_account(
-                patron
-            )
-            credential.credential = remote_identifier
-            credential.expires = None
+            try:
+                remote_identifier = self._find_or_create_remote_account(
+                    patron
+                )
+                credential.credential = remote_identifier
+                credential.expires = None
+            except CirculationException:
+                self.log.exception("Failed to create RBDigital User. Collection: %s", self.collection.name)
```

Comment:
Is it reasonable to drop the exception in this case? The argument for is that we're immediately going to try to do something with (potentially) no credential and (if so) get a "real" error. Is that right?

Reply:
Yes. That is correct. I added this to avoid creating credentials with no value in the DB.

Reply:
I reworked this so its now swallowing the exception anymore.